### ISSUE TITLE
Add default local replica API keys for Ankr L2 providers

### ISSUE THREAD

**Issue by rvanasa** [internet-computer-protocol/evm-rpc-canister#255]
[Relevant forum topic](https://forum.dfinity.org/t/error-calling-eth-sendrawtransaction-on-arbitrum-ankr/34208).

**Comment by rvanasa** [comment]
Closed as no longer relevant after #244.
